src/components/calculate.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
```

Comment:
- [ ] The requirement states that this file name should be created as `Calculator.js`, not `calculate.js`. There is a need for us to make the project requirements to be the first thing, then our own can be secondary.